script.js

```diff
@@ -0,0 +1,206 @@
+const store = localStorage.getItem('lang');
```

Comment:
not very logical name for vairable which contains language value, examples of more preferable names: "lang", "savedLang" etc